### ISSUE TITLE
KVM version of GNS3, adding a NAT node says "NAT interface virbr0 is missing, please install libvirt", although libvirt is installed and running

### ISSUE THREAD

**Issue by mridhop** [GNS3/gns3-gui#3404]
**Before you start**
Please open an issue only if you suspect there is a bug or any problem with GNS3. Go to https://gns3.com/community for any other questions or for requesting help with GNS3.

You may also post this issue directly on the GNS3 server repository if you know the bug comes from the server: https://github.com/GNS3/gns3-server/issues/new

**Describe the bug**
Hello, so, I'm not able to insert a NAT node to my topology, it says "NAT interface virbr0 is missing, please install libvirt". The thing is I already have libvirt and the service is already running, also my user is already in the 'kvm' and 'libvirt' group.

I tried starting gns3 in 2 ways, one is using the `start-gns3vm.sh` script that you get inside the GNS3.VM.KVM.<version>.zip file, the second is with virt-manager, both resulting to the same problem.

I already tried all suggestions I found on the internet, such as:
1. Messing with `virsh`, tried undefining and redefining the 'default' config
2. Changing firewall from ufw to firewalld
3. Restarting libvirtd service

Also, selecting the "Test | Check Internet connection" in the GNS3 menu shows that connection to internet is OK.

**GNS3 version and operating system (please complete the following information):**
 - OS: Linux (Arch Linux)
 - GNS3 version: Tried 2.2.34, 2.2.26, 2.2.19 (all are the KVM version)
 - Any use of the GNS3 VM or remote server (ESXi, bare metal etc.): No

**To Reproduce**
1. Start a version of gns3 kvm
2. Go to the web ui
3. Add a NAT node, get the error "NAT interface virbr0 is missing, please install libvirt"

Sorry for my bad English, help is so much appreciated! Thanks in advance.

**Additional Notes**
I'm able to run and add a NAT node to gns3-gui natively and through VMWare perfectly, it's just I can't find a way to do it through QEMU/KVM.

**Comment by PranavBhattarai** [comment]
I think it would have huge help and time savior if there was an official KVM/QEMU image sooner or later.
We don't even have an official blog on how to set up KVM/QEMU in VirtManager.
Once have to go through many hacks in order to use VirtManager with KVM/QEMU image.

Since VirtManager is the best free and opensource alternative through which users can try KVM/QEMU, instead of fully depending upon VMware (non-free). It's good to have an alternative.

@grossmj Any thoughts on this? Shall I create a new issue "feature requesting" KVM/QEMU VirtManager support?!

**Comment by josephmhiggins** [comment]
@PranavBhattarai,
the contributing.md says you should get feedback from the gns3 community first:
https://github.com/GNS3/gns3-gui/blob/master/CONTRIBUTING.md
the documentation to guide you to create a suggest, or enhancement, or feature request is here:
https://github.com/GNS3/gns3-gui/blob/master/.github/ISSUE_TEMPLATE/gns3-feature-request.md

Just put the suggestion in on github, or go to the gns3 community board and ask for feedback on your idea and I may, or may not, supply a long rendition of the history of gns3 vis-a-vis virt-manager.

**Comment by PranavBhattarai** [comment]
@josephmhiggins I was thinking about deleting the above comment when it receive no response.
I know technically this is the wrong place to ask for Virt-Manager support. I rather have created a new issue regarding it or a question in GNS3 Community/Forum.
Thank you for reminding this. And sorry for hijacking the thread.

**Comment by grossmj** [comment]
That's strange, we use psutil to find interfaces.

Please can you try with Python directly? This command should work:

```
python3 -m pip install psutil && python3 -c "import psutil; print(psutil.net_if_addrs().keys())"
```

**Comment by grossmj** [comment]
@PranavBhattarai have you tried the command I posted above?

**Comment by PranavBhattarai** [comment]
@grossmj Sorry I was not helpful on this issue, because I didn't originally create this issue. I just asked a question when I saw KVM.
However, I tried to reproduce the issue but failed in Step 2 which OP posted. i.e.
> Go to the web UI

I created a simple topology and selected **Tools > Web UI - beta**.
![image](https://user-images.githubusercontent.com/32763112/216755051-c2dc9a29-3f35-4798-a4d5-f77793d0cfef.png)
It launched my default browser (Firefox) and asked me for permission (which I gave) but **didn't show anything**.
No topology so far. I tried disabling Firefox's Enhanced Tracking, but it didn't any difference. 

Since the issue is already, closed, and the OP is also not responding, we can assume, the problem doesn't exist anymore.
```
Fedora Workstation 37
GNS3 version: 2.2.34
GNS3VM using VirtManager
```
